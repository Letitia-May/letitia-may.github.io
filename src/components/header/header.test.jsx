import React from 'react'
import { render } from '@testing-library/react'
import Header from './header'

describe(Header, () => {
  test('renders the Header with name and job title', () => {
    const { getByText } = render(<Header />)

    expect(getByText('Chloe')).toBeInTheDocument()
    expect(getByText('Web dev')).toBeInTheDocument()
  })
})
