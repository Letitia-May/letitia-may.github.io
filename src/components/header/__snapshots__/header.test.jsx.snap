// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="header"
>
  <div
    className="header_content"
  >
    <div
      className="link"
    >
      <a
        href="/"
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        <p
          className="main_title"
        >
          Chloe
        </p>
        <p
          className="subtitle"
        >
          Web dev
        </p>
      </a>
    </div>
  </div>
</div>
`;
